primitives/polkadot-core/src/parachains.rs

```diff
@@ -16,6 +16,7 @@
 
 //! Primitives of polkadot-like chains, that are related to parachains functionality.
 //!
+//! TODO:check-parameter - fix comment or can we import `ParaId` from polkadot now? does it make sense?
 //! Even though this (bridges) repository references polkadot repository, we can't
```

Comment:
IDK how it'll behave when we have subtree in cumulus. Do bridge hubs runtimes always reference the polkadot master? If so, then I guess we may fix it. Let's maybe file an issue for that now?

Reply:
ok, thank you, just asking,

I will just change "polkadot" to "Cumulus" here. We dont need to introduce polkadot dependencies, we just use them in `bridge-runtime-common` where it is ok.

Cumulus master references master (polkadot, substrate).
Cumulus release branches references release branches (polkadot, substrate).

Anyway, there is a plan to move parity-bridges-common to monorepo also (maybe partially instead of subtree), but not now, maybe later.